docs/src/documentation/03-components/05-input/inputgroup/03-accessibility.mdx

```diff
@@ -0,0 +1,124 @@
+---
+title: Accessibility
+redirect_from:
+  - /components/inputgroup/accessibility/
+---
+
+## Accessibility
+
+The InputGroup component has been designed with accessibility in mind, providing a fieldset structure that groups related form controls with proper semantic markup and screen reader support.
+
+### Accessibility Props
+
+| Name           | Type     | Description                                                                                   |
+| :------------- | :------- | :-------------------------------------------------------------------------------------------- |
+| ariaLabel      | `string` | Provides an accessible label for the fieldset when no visible `label` is provided.            |
+| ariaLabelledby | `string` | References the ID of an element that labels the fieldset.                                     |
+| label          | `string` | Provides a visible legend for the fieldset that also serves as the accessible name.           |
+| required       | `bool`   | When `true`, applies `aria-required="true"` to indicate at least one field must be completed. |
+
+### Automatic Accessibility Features
+
+- The component automatically manages ARIA attributes:
+  - `aria-required="true"` when the `required` prop is set
+  - `aria-describedby` on all child elements when InputGroup has `error` or `help` content and the tooltip is shown
+  - Unique feedback ID generation for error/help content association
```

Comment:
I don't know if I would mention this as it's only internal, no one can access its value, so it might be confusing.